src/Util/Asserts.h

```diff
@@ -28,25 +28,27 @@
         NSCAssert(0, @"Assertion failed: %s", CONVERT_EXPR_TO_STRING(X));                                              \
     }
 
-#define OWSFail(X)                                                                                                     \
+#define OWSFail(message, ...)                                                                                          \
```

Comment:
I don't think so. I think NS_FORMAT_FUNCTION is used to decorate the declarations of  functions and methods. It's not for macros.  For example, in this case, the macro will be preprocessed to `[NSString stringWithFormat:...]` whose declaration has `NS_FORMAT_FUNCTION`: 
```
+ (instancetype)stringWithFormat:(NSString *)format, ... NS_FORMAT_FUNCTION(1,2);
```